freediscovery/data/ds_001/raw/0.7.6.28635.txt

```diff
@@ -1,4 +1,4 @@
-﻿From:           Eric Bass </o=teneo-test/ou=first administrative group/cn=recipients/cn=bass_eric>
+From:           Eric Bass </o=teneo-test/ou=first administrative group/cn=recipients/cn=bass_eric>
```

Comment:
ok, so after reverting this file here is the error:

`UnicodeEncodeError: 'charmap' codec can't encode character '\ufeff' in position 0: character maps to <undefined>`

Looks like adding `.replace(u'\ufeff','')` solves the problem
solution taken from [here](http://stackoverflow.com/questions/17912307/u-ufeff-in-python-string) (see the 2nd comment under the question)